zerver/webhooks/bitbucket2/doc.md

```diff
@@ -6,21 +8,42 @@ For the old-style Bitbucket webhooks used by Bitbucket Enterprise,
 click [here](./bitbucket), and for the new-style webhooks used by
 Bitbucket Server click [here](./bitbucket3).
 
+{start_tabs}
+
 1. {!create-channel.md!}
 
 1. {!create-an-incoming-webhook.md!}
 
-1. {!generate-integration-url.md!}
+1. {!generate-webhook-url-basic.md!}
+
+1. On your repository's web page, select **Settings**.
 
-    {!git-webhook-url-with-branches.md!}
+1. Select **Webhooks**, and then select **Add webhook**.
 
-1. On your repository's web page, click on **Settings**. Select
-   **Webhooks**, and click **Add webhook**.
+1. Set **Title** to a title of your choice, such as `Zulip`.
 
-1. Set **Title** to a title of your choice, such as `Zulip`. Set **URL**
-   to the URL constructed above, and check the **Active** checkbox. Select
-   the **Triggers** you'd like to be notified about, and click **Save**.
+1. Set **URL** to the URL generated above, and toggle the **Active**
+   checkbox.
+
+1. Select the **Triggers** you'd like to be notified about, and select
+   **Save**.
```

Comment:
So the structure of these instructions was previously designed to have:
- A single step for "Navigate to the right forum".
- A single step for "Fill out the form".

Then benefit of this is that I think it feels less intimidating/overwhelming to see a list of 5 steps than a like of 9 where most of the steps are a single click or field to fill out. I think for the "fill out the form" steps here, I'd prefer to read something like:

> Fill out the form, using the URL generated above as the **URL**, a **Title** like "Zulip" (it's only used for this settings panel), and the **Triggers** that you'd like to trigger notifications.

We could also try using bulleted lists.

Let's discuss in "#documentation" these patterns for a potential follow-up.

Reply:
Yeah, grouping as such makes sense. Making those updates.